pages/common/pup.md

```diff
@@ -0,0 +1,27 @@
+# pup
+
+> HTML parsing tool.
+
+- Transform raw HTML file into a cleaned, indented, and colored format:
+
+`cat {{index.html}} | pup --color`
+
+- Filter HTML by element tag name:
```

Comment:
With the way this and further examples on this page are worded, it feels like the HTML it takes in isn't 'raw'? Perhaps the first example needs rewording - I'm not sure.

Reply:
Why not ? By raw HTML, I think its meant that whitespace is removed and the tags are not indented properly. The wording looks okay to me.

Reply:
maybe `unformatted` would be clearer?

Reply:
Ah, ok. Let's leave it then :-)